dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/exception/UninitialisedVariableException.java

```diff
@@ -0,0 +1,8 @@
+package com.dat3m.dartagnan.parsers.program.exception;
+
+public class UninitialisedVariableException extends RuntimeException {
```

Comment:
You might wanna extend from `MalformedProgramException`. I would say an uninitialized variable is also a case of wrongly formed program.